File: test/aidbox_sdk/snapshots/typescript/hl7-fhir-r4-core/ResearchSubject.ts
import { Period } from "./Period";
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";
import { Identifier } from "./Identifier";

export type ResearchSubject = DomainResource & {
    study: Reference;
    period?: Period;
    status: string;
    consent?: Reference;
    actualArm?: string;
    identifier?: Identifier[];
    individual: Reference;
    assignedArm?: string;
};

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/ResearchSubject.py
from typing import Optional, List
from base import Period
from base import DomainResource
from base import Reference
from base import Identifier

class ResearchSubject(DomainResource):
    study: Reference
    period: Optional[Period] = None
    status: str
    consent: Optional[Reference] = None
    actual_arm: Optional[str] = None
    identifier: Optional[List[Identifier]] = None
    individual: Reference
    assigned_arm: Optional[str] = None